### ISSUE TITLE
Is there an issue with the code for calculating the loss function

### ISSUE THREAD

**Issue by gczzcc** [JiehongLin/Self-DPDN#15]
I noticed that there might be an issue with the code for calculating the final pose loss function in the file located at 'DPDN-Pytorch1.9.0-Cuda11.2/model/losses.py'.
In the source code:
def PoseDis(r1, t1, s1, r2, t2, s2):
    '''
    r1, r2: b*3*3
    t1, t2: b*3
    s1, s2: b*3
    '''
    dis_r = torch.mean(torch.norm(r1 - r2, dim=1))
    dis_t = torch.mean(torch.norm(t1 - t2, dim=1))
    dis_s = torch.mean(torch.norm(s1 - s2, dim=1))

    return dis_r + dis_t + dis_s
Both r1 and r2 are tensors with shape b*3*3. Therefore, should dis_r = torch.mean(torch.norm(r1 - r2, dim=1)) be modified to dis_r = torch.mean(torch.norm(r1 - r2, dim=(1,2)))?
